web/src/app/core/services/project.service.ts

```diff
@@ -46,6 +54,26 @@ export class ProjectService {
       );
   }
 
+  // This function delete the project
+  showDeleteDialog(projectUid: string): void {
+    let dialogConfig: MatDialogConfig = new MatDialogConfig();
+    dialogConfig = {
+      width: '500px',
+      data: {
+        title: 'Delete Project',
+        content: 'Are you sure you want to delete?',
+      },
+    };
+    this.dialogRef = this.dialog.open(DialogConfirmationComponent, dialogConfig);
+    this.dialogRef.afterClosed()
+      .subscribe((result: boolean) => {
```

Comment:
Good idea to move this here to be reusable. I think `subscribe` should not really be in the `service`, so that the component consuming it can decide what to do